fdbclient/MultiVersionTransaction.actor.cpp

```diff
@@ -1586,8 +1628,15 @@ void MultiVersionDatabase::DatabaseState::updateDatabase(Reference<IDatabase> ne
 			    .detail("ClusterFilePath", clusterFilePath);
 		}
 	}
-
-	dbVar->set(db);
+	if (db.isValid() && dbProtocolVersion.present()) {
```

Comment:
I think we need to check if the version supports database sharing, otherwise we will get unsupported_operation error here